docs/sources/alerting/fundamentals/alert-rules/_index.md

```diff
@@ -1,4 +1,9 @@
 ---
+aliases:
+  - ../fundamentals/data-source-alerting/
+  - ../fundamentals/alert-rules/alert-instances/
+  - ../fundamentals/alert-rules/recording-rules/
+  - ../fundamentals/alert-rules/alert-rule-types/
```

Comment:
Could we also add the comments here for the future.

Reply:
addressed by https://github.com/grafana/grafana/pull/84248/commits/1133f2d73d48d096fedd2b7f76eec40923e86344